### DISCUSSION TITLE
Suggestion: license tag in the default mod template

### DISCUSSION THREAD

**Discussion by Anixx** [ramensoftware/windhawk-mods#494] [General]
Many people do not bother with specifying license of their mods. Adding the default license as a tag to the default mod template will increase the percent of third-party mods with clear license specification.

**Comment by m417z** [comment]
I got some inspiration from user scripts, and [they don't have it](https://wiki.greasespot.net/Metadata_Block). But yes, I agree that it can be a good idea, I'll consider it for future versions.

**Comment by Anixx** [comment]
If we talk about a RPM packages in Linux, it is a required field in the .spec file, and even the format of the license name is standardized.
